twisted/names/dns.py

```diff
@@ -2637,23 +2637,23 @@ def toStr(self):
     @classmethod
     def _fromMessage(cls, message):
         """
-        Construct and return a new L(_EDNSMessage} whose attributes and records
```

Comment:
No, change this back.  There is one bug on this line but you didn't fix it.
This should be:
L{_EDNSMessage}